### PR TITLE
Merge panda

### PR DESCRIPTION
**Pull request by cremebrule**
Added the following panda envs:
[`Panda`], `PandaLift`, `PandaStack`, `PandaNutAssembly`, `PandaPickPlace`

Added gripper `PandaGripper`

Added support for panda ik_control (+ necessary pybullet files)

A few TODOs remaining to clarify desired protocol (see TODOs for specifics).

### CODE REVIEW

File: robosuite/environments/sawyer.py [hunk 1]
```diff
@@ -339,6 +339,7 @@ def _right_hand_quat(self):
         """
         Returns eef quaternion in base frame of robot.
         """
+        print("right_hand_orn: {}".format(self._right_hand_orn))
```
Comment (yukezhu) [inline robosuite/environments/sawyer.py]: why is this line added?

File: robosuite/controllers/sawyer_ik_controller.py [hunk 2]
```diff
@@ -71,6 +71,7 @@ def get_control(self, dpos=None, rotation=None):
 
         # Compute new target joint positions if arguments are provided
         if (dpos is not None) and (rotation is not None):
+
```
Comment (yukezhu) [inline robosuite/controllers/sawyer_ik_controller.py]: why are the lines of white space added?

Comment (cremebrule) [reply robosuite/controllers/sawyer_ik_controller.py]: originally had debugging print statements

File: robosuite/environments/sawyer.py (hunk 1 above)
Comment (cremebrule) [reply robosuite/environments/sawyer.py]: debugging statement

**Comment by yukezhu** [comment]
image

The gripper frame axes are shown on the Panda arm (see above), which might cause issue for vision-based policy learning. @cremebrule can you hide them from the rendering?

File: .gitignore [hunk 3]
```diff
@@ -112,3 +112,6 @@ mjkey.txt
 .idea
 
 .pytest_cache/
+
```
Comment (yukezhu) [inline .gitignore]: Clean up this test script

File: robosuite/scripts/demo_device_ik_control.py [hunk 4]
```diff
@@ -87,7 +87,8 @@
         env.render()
 
         # rotate the gripper so we can see it easily
-        env.set_robot_joint_positions([0, -1.18, 0.00, 2.18, 0.00, 0.57, 1.5708])
+        #env.set_robot_joint_positions([0, -1.18, 0.00, 2.18, 0.00, 0.57, 1.5708])
+        env.set_robot_joint_positions([0, np.pi / 16.0, 0.00, -np.pi / 2.0 - np.pi / 3.0, 0.00, np.pi - 0.2, -np.pi/4])
```
Comment (yukezhu) [inline robosuite/scripts/demo_device_ik_control.py]: This new initial pose doesn't work for Sawyer. Can you add a if-condition to use the previous pose for Sawyer and this new pose for Panda?

File: robosuite/scripts/demo_device_ik_control.py [hunk 5]
```diff
@@ -109,8 +110,13 @@
             dquat = T.mat2quat(drotation)
 
             # map 0 to -1 (open) and 1 to 0 (closed halfway)
-            grasp = grasp - 1.
+            #grasp = grasp - 1.
```
Comment (yukezhu) [inline robosuite/scripts/demo_device_ik_control.py]: remove line

File: robosuite/scripts/demo_device_ik_control.py [hunk 6]
```diff
@@ -109,8 +110,13 @@
...
+            if grasp:
+                grasp = -1
+            else:
+                grasp = 1
 
             action = np.concatenate([dpos, dquat, [grasp]])
+            print("action: {}".format(action))
```
Comment (yukezhu) [inline robosuite/scripts/demo_device_ik_control.py]: remove line

File: robosuite/scripts/demo_device_ik_control.py (hunk 5 above)
Comment (cremebrule) [reply robosuite/scripts/demo_device_ik_control.py]: done

File: robosuite/scripts/demo_device_ik_control.py (hunk 4 above)
Comment (cremebrule) [reply robosuite/scripts/demo_device_ik_control.py]: done

File: robosuite/scripts/demo_device_ik_control.py (hunk 6 above)
Comment (cremebrule) [reply robosuite/scripts/demo_device_ik_control.py]: done
